modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements_controller.rb

```diff
@@ -1,46 +1,37 @@
 # frozen_string_literal: true
 
+require 'json_schema/json_api_missing_attribute'
 require 'appeals_api/form_schemas'
 
 class AppealsApi::V1::DecisionReviews::NoticeOfDisagreementsController < AppealsApi::ApplicationController
   include AppealsApi::JsonFormatValidation
 
   skip_before_action(:authenticate)
   before_action :validate_json_format, if: -> { request.post? }
+  before_action :validate_json_schema, only: %i[create validate]
   before_action :new_notice_of_disagreement, only: %i[create validate]
   before_action :find_notice_of_disagreement, only: %i[show]
 
   FORM_NUMBER = '10182'
+  MODEL_ERROR_STATUS = 422
   HEADERS = JSON.parse(
     File.read(
       AppealsApi::Engine.root.join('config/schemas/10182_headers.json')
     )
   )['definitions']['nodCreateHeadersRoot']['properties'].keys
 
   def create
-    if @notice_of_disagreement.save
-      AppealsApi::NoticeOfDisagreementPdfSubmitJob.perform_async(@notice_of_disagreement.id)
-      render_notice_of_disagreement
-    else
-      render_model_errors
-    end
+    @notice_of_disagreement.save
+    AppealsApi::NoticeOfDisagreementPdfSubmitJob.perform_async(@notice_of_disagreement.id)
+    render_notice_of_disagreement
   end
 
   def show
     render_notice_of_disagreement
   end
 
   def validate
-    if @notice_of_disagreement.valid?
-      render json: {
-        data: {
-          type: 'noticeOfDisagreementValidation',
-          attributes: { status: 'valid' }
-        }
-      }
-    else
-      render_model_errors
-    end
+    render json: validation_success
```

Comment:
Are we not concerned with rendering validation errors from `/validate` after this?

Reply:
Nevermind, I see the callback that renders the json directly.